config/manifests/bases/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: operator-certification-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: OperatorPipeline is the Schema for the operatorpipelines API
+      displayName: Operator Pipeline
+      kind: OperatorPipeline
+      name: operatorpipelines.certification.redhat.com
+      version: v1alpha1
+  description: An Operator for setting up Operator Certification CI Pipeline
+  displayName: Operator Certification Operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - certification
+  - pipeline
+  links:
+  - name: Operator Certification Operator
+    url: https://github.com/redhat-openshift-ecosystem/operator-certification-operator
+  maintainers:
+  - email: jomckenz@redhat.com
+    name: jomkz
+  maturity: alpha
```

Comment:
may want to add `minKubeVersion: 1.21.0` - per [pipelines docs](https://github.com/redhat-openshift-ecosystem/certification-releases/blob/main/4.9/ga/ci-pipeline.md#what-youll-need-before-you-start) we should require 4.8/1.21.0 or above.

Reply:
Since the doc states 4.8 is recommended and mentions tested with cli 4.7 - should we be strict and require 1.21.0,  or can we require 1.18.0 (OpenShift 4.7) ?

Reply:
Yeah, the docs say " (recommended version 4.8 or above)" so I am not sure there is a hard requirement for 1.21.0 per se. @tonytcampbell Do you have any thoughts on this? Is it that we just haven't tested on anything below 4.8/1.21?